### ISSUE TITLE
Download output of different kernel versions 

### ISSUE THREAD

**Issue by phuc16102001** [Kaggle/kaggle-cli#442]
**Problem:**
Because I have a notebook to crawl data from external sources, and I have scheduled it to run daily. 
So I want to download the kernel's output in many different versions, however, Kaggle API only is able to download the last one.

**Feature propose:** Add a parameter to kernel output APIs for the version

**Comment by phuc16102001** [comment]
If anyone faces the same issue, you can consider using my [Kaggle output downloader](https://github.com/phuc16102001/kaggle-output-downloader) to automate the download process.

**Comment by qrdlgit** [comment]
really need this, eg:  something where we just append the scriptVersionId=# would do the trick

**Comment by phuc16102001** [comment]
@qrdlgit 
Maybe you can checkout my repository & change it if you needed.
You can create a PR on it & I will review it.

**Comment by qrdlgit** [comment]
yeah I saw that.  very cool you built that for sure.   I try to avoid pulling in all those external deps when engineering as they inevitably lead to maintenance nightmares, but I absolutely get why you and probably I will be forced to use it.

What annoys me is that Kaggle reached out to me and asked to do a phone interiew about the kaggle api.  I mean.. .really?   just look at the git... all the feedback in the world you need.

I find the radio silence from kaggle kinda arrogant at times.

**Comment by phuc16102001** [comment]
Yes I agree that, because the solution I used is integration test anyway, so it will be broken if the Kaggle change a small thing. I hope that the Kaggle API will provide us a feature on this 😢, but they still not, so it is the only current solution.

Previously, I have a notebook which is scheduled to run daily & it store the output data day by day. So I cannot manually download all of them, that's why I must create that ;;-;;

**Comment by aditya-ihx** [comment]
Can't believe this is still not fixed 😢 I want the old kernel version not the latest one 😢

**Comment by vohuutridung** [comment]
> Can't believe this is still not fixed now😢
